Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```
@@ -63,7 +63,7 @@
     - id: ClothingHeadCage # backmen
     - id: HandheldReinforcementCentCom # backmen
 #    - id: ClothingEyesGlassesCommand # Backmen - Because dublicate (56)???
-
+     
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)